packages/@ember/-internals/deprecations/index.ts

```diff
@@ -129,6 +129,16 @@ export const DEPRECATIONS = {
       enabled: '5.10.0',
     },
   }),
+  DEPRECATE_ARRAY_PROTOTYPE_EXTENSIONS: deprecation({
+    id: 'deprecate-array-prototype-extensions',
+    url: 'https://deprecations.emberjs.com/id/deprecate-deprecate-array-prototype-extensions',
```

Comment:
This points to https://github.com/ember-learn/deprecation-app/pull/1192, which we also need to get merged.